sdk/appconfiguration/tests.yml

```diff
@@ -8,9 +8,22 @@ stages:
         - name: azure-data-appconfiguration
           groupId: com.azure
           safeName: azuredataappconfiguration
+      TimeoutInMinutes: 90
```

Comment:
We need more contents here. Refs: https://github.com/Azure/azure-sdk-for-java/pull/18916/files#diff-a38d5bba40ecff60e025723b70c9a682e464bcadf5e87e6571a641ca8659dee3L7